core/lock.c

```diff
@@ -74,6 +74,9 @@ int pthread_mutexattr_setprotocol (pthread_mutexattr_t *__attr,
 struct uwsgi_lock_item *uwsgi_lock_fast_init(char *id) {
 
 	pthread_mutexattr_t attr;
+#ifdef EOWNERDEAD
```

Comment:
Sure. Personally I don't like declaring variables all over the place but I suppose it makes sense here.